src/client/pages/ConsentsCommunication.tsx

```diff
@@ -57,23 +59,30 @@ export const ConsentsCommunication = ({
       bgColor={CONSENTS_MAIN_COLOR}
     >
       <ConsentsContent>
-        <h2 css={[heading, autoRow()]}>Thank you for registering</h2>
-        <p css={[text, autoRow(consentsParagraphSpanDef)]}>
+        <h2 css={[heading, greyBorderTop, autoRow()]}>
+          Thank you for registering
+        </h2>
+        <p css={[text, p, autoRow(consentsParagraphSpanDef)]}>
           Would you like to join our mailing list to stay informed and up to
           date with all that the Guardian has to offer?
         </p>
-        <div
-          css={[communicationCardContainer, autoRow(communicationCardSpanDef)]}
-        >
-          {consents.map((consent) => (
-            <CommunicationCard
-              key={consent.id}
-              title={consent.name}
-              body={consent.description}
-              value={consent.id}
-              checked={!!consent.consented}
-            />
-          ))}
+        <div css={[autoRow()]}>
+          <div
+            css={[
+              communicationCardContainer,
+              autoRow(communicationCardSpanDef),
+            ]}
+          >
+            {consents.map((consent) => (
+              <CommunicationCard
+                key={consent.id}
+                title={consent.name}
+                body={consent.description}
+                value={consent.id}
+                checked={!!consent.consented}
```

Comment:
i think the `image` prop is missing from here, which is why the image isn't showing up for the consent